### DISCUSSION TITLE
Skipping photos based on name instead of hash

### DISCUSSION THREAD

**Discussion by andrewbanchich** [ente-io/ente#3070] [Enhancements]
### Description

When uploading I see "Skipped these as there are files with matching names in the same album".

Could Ente compare a hash of the contents + metadata instead of just looking at the name?

### Version

1.7.3

### What product are you using?

Ente Photos

### What platform are you using?

Desktop - Linux

**Comment by mnvr** [comment]
We already look at both the hash and the name of the file when skipping. The copy might need tweaking.

**Reply by mnvr** [reply]
I've updated the copy in https://github.com/ente-io/ente/pull/3072. Changes will be available in the next release. Will close this thread, but please reopen if I misunderstood something.

**Reply by andrewbanchich** [reply]
Awesome, thanks!
